.github/CODEOWNERS

```diff
@@ -1277,13 +1299,29 @@ x-pack/plugins/cloud_integrations/cloud_full_story/server/config.ts @elastic/kib
 /x-pack/test_serverless/api_integration/test_suites/search/cases/ @elastic/response-ops
 /x-pack/test_serverless/api_integration/test_suites/observability/cases/ @elastic/response-ops
 /x-pack/test_serverless/api_integration/test_suites/security/cases/ @elastic/response-ops
+/x-pack/test_serverless/functional/test_suites/**/screenshot_creation/ @elastic/response-ops
```

Comment:
I think response ops only owns the `response_ops_docs` under `screenshot_creation`, not the `ml_docs` or `transform_docs` folders

Reply:
Good point, I didn't see `ml_docs` or `transform_docs` folder under `x-pack/test_serverless` but it might be added similar to stateful tests. Updated 6a5ca78ca1a11c5c6ab292c7ca08c23ae4c28552